src/canvas/canvas.class.ts

```diff
@@ -834,11 +834,14 @@ export class SelectableCanvas<
     if (!target) {
       return;
     }
-    let pointer = this.getPointer(e);
-    if (target.group) {
-      // transform pointer to target's containing coordinate plane
-      pointer = sendPointToPlane(pointer, target.group.calcTransformMatrix());
-    }
+    const pointer = target.group
+      ? // transform pointer to target's containing coordinate plane
+        sendPointToPlane(
+          this.getPointer(e),
+          undefined,
+          target.group.calcTransformMatrix()
+        )
+      : this.getPointer(e);
```

Comment:
this is a code style comment.
I do like to optimize for the most common case rather than not.
putting the special case of the group in front of the normal one is more distracting.

Reply:
point taken